src/test/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbeTest.java

```diff
@@ -0,0 +1,61 @@
+package io.jenkins.pluginhealth.scoring.probes;
+
+import java.time.ZonedDateTime;
+
+import io.jenkins.pluginhealth.scoring.model.Plugin;
+import io.jenkins.pluginhealth.scoring.model.ProbeResult;
+import io.jenkins.pluginhealth.scoring.model.ResultStatus;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
I don't see any mock, why do you need this?

Reply:
I initially tried to mock the Classes and got a little confused then I thought "Why do I even need to mock if I can use demo values as per the test case". So I took a shortcut without realizing that it isn't an ideal way to write unit tests and didn't consider the points you raised about the internet being down.

And then I forgot to delete this annotation.